index.js

```diff
@@ -3,6 +3,29 @@ const fs = require('fs');
 
 module.exports = {
   improve: '@apostrophecms/piece-type',
+  batchOperations (self) {
+    return {
+      add: {
+        export: {
+          label: 'Export',
```

Comment:
After an initial test it seems like modules like this that only `improve` other modules may not be able to have their own i18n namespace and localization JSON files. It's something we'll need to figure out soon, but it might require a core improvement.